README.md

```diff
@@ -1,29 +1,20 @@
 # Composer Asset Compiler
 
-_Composer plugin that install dependencies and compile assets based on configuration._
+_Composer plugin that installs dependencies and compiles assets based on configuration._
 
 ![PHP Quality Assurance](https://github.com/inpsyde/composer-asset-compiler/workflows/PHP%20Quality%20Assurance/badge.svg)
 
 ----
 
 ## What is this / Why bother
 
-When working with WordPress and Composer for the whole site, we use Composer to pull together
-plugins an themes. Those will often have **frontend assets** (as in: javascript, CSS...), and many
-of them will use **frontend assets processing** (an is: Webpack, Grunt, Gulp...).
+When working with WordPress and Composer for the whole site, we use Composer to pull together plugins and themes. These often have **frontend assets** (like JavaScript, CSS, etc.), and many of them use **frontend assets processing* (like webpack, Grunt, gulp, etc.).
```

Comment:
Thank you for your feedback.

I think line breaks in text files should be set by the processing/display tools, not by the content itself. The diff view above is a good example, which has "illogical" line breaks after the words "together" and "many" because the line length of this discussion view is shorter than that of the text.

However, if you insist, I will change this.

Reply:
I don't have strong opinion, but markdown is not "text file" more than an HTML is a text file
Both HTML and Markdown are a markup format, and both formats consider line breaks unrelevant.

Just like an HTML file, when rendered, looks the same regardless the line breaks, the same happens to a Markdown file.

And just like we try to use short lines in HTML files for readibility during editing, I feel we should do the same for markdown files, and for the same reason.